demo/opae-nlb-demo/Dockerfile

```diff
@@ -15,7 +15,7 @@ ARG CLEAR_LINUX_VERSION=
 RUN swupd update --no-boot-update ${CLEAR_LINUX_VERSION} && \
     swupd bundle-add wget c-basic devpkg-json-c devpkg-util-linux devpkg-hwloc doxygen Sphinx patch
 # Fetch dependencies and source code
-ARG OPAE_RElEASE=1.3.2-1
+ARG OPAE_RElEASE=1.4.0-1
```

Comment:
nit: could you please rename it to `OPAE_RELEASE`?

Reply:
done